server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -450,6 +449,7 @@ public class ConfigurationManagerImpl extends ManagerBase implements Configurati
     private Set<String> overprovisioningFactorsForValidation;
     public static final String VM_USERDATA_MAX_LENGTH_STRING = "vm.userdata.max.length";
 
+    public static final String HOST_RESERVED_MEM_MB_STRING = "host.reserved.mem.mb";
```

Comment:
We could use `HOST_RESERVED_MEM_MB.key()` instead of declaring a constant with it.

Reply:
Done 👍